src/hotspot/share/compiler/compilerDefinitions.inline.hpp

```diff
@@ -25,6 +25,8 @@
 #ifndef SHARE_COMPILER_COMPILERDEFINITIONS_INLINE_HPP
 #define SHARE_COMPILER_COMPILERDEFINITIONS_INLINE_HPP
 
+#include "c1/c1_Compiler.hpp"
+#include "opto/c2compiler.hpp"
```

Comment:
These should be also under  COMPILER*_PRESENT macros.

Reply:
Right! Fixed. Thanks @vnkozlov.